run_helper_scripts/2019-21 processing.py

```diff
@@ -6,6 +6,9 @@
 import pathlib
 
 ##############
+"""
+
```

Comment:
Please fill in this docstring and put it at the very start of the docstring before all the imports